hs_core/templates/pages/create-resource.html

```diff
@@ -3,153 +3,648 @@
+                        <li>
+                            <a data-toggle="tooltip" data-placement="right"
+                               title="A shortcut (or reference) to an external web application that functions with HydroShare data resources."
+                               data-value="ToolResource">
+                                <img src="{{ STATIC_URL }}img/resource-icons/webapp48x48.png"
+                                     alt="Web App Resource Icon" class="resource-type-icon"/>
+                                <div class="resource-type-name">&nbsp;Web App</div>
+                            </a>
+                        </li>
+                        <li role="separator" class="divider"></li>
+                        <li>
+                            <a data-toggle="tooltip" data-placement="right"
+                               title="Defines the input and output data for a generic hydrological model, for a specific time and place."
+                               data-value="ModelInstanceResource">
+                                <img src="{{ STATIC_URL }}img/resource-icons/modelinstance48x48.png"
+                                     alt="Model Instance Resource Icon" class="resource-type-icon"/>
+                                <div class="resource-type-name">&nbsp;Model Instance</div>
+                            </a>
+                        </li>
+                        <li>
+                            <a data-toggle="tooltip" data-placement="right"
+                               title="A specific case of the Model Instance resource that applies to USGS MODFLOW 2000 and later versions."
+                               data-value="MODFLOWModelInstanceResource">
+                                <img src="{{ STATIC_URL }}img/resource-icons/modflow48x48.png"
+                                     alt="Collection Resource Icon" class="resource-type-icon"/>
+                                <div class="resource-type-name">&nbsp;MODFLOW Model Instance</div>
+                            </a>
+                        </li>
+                        <li>
+                            <a data-toggle="tooltip" data-placement="right"
+                               title="A specific case of the Model Instance resource that applies to the Soil and Water Assessment Tool."
+                               data-value="SWATModelInstanceResource">
+                                <img src="{{ STATIC_URL }}img/resource-icons/swat48x48.png"
+                                     alt="SWAT Model Instance Resource Icon" class="resource-type-icon"/>
+                                <div class="resource-type-name">&nbsp;SWAT Model Instance</div>
+                            </a>
+                        </li>
+                        <li>
+                            <a data-toggle="tooltip" data-placement="right"
+                               title="Describes the software component of a generic model within the water domain."

```

Comment:
@mauriel133 Similar tooltip construct as above, but I'll leave it up to you whether you want to change it or not. Perhaps quickly run by @horsburgh for the summary tooltip text for each resource type?